apps/api/src/database/schema/util.ts

```diff
@@ -1,10 +1,10 @@
 import { numeric, varchar } from "drizzle-orm/pg-core";
 
 /**
- * PG column for amount of tokens in humanese
+ * PG column for amount of tokens in wei (BigInt representation)
  */
 export function tokenAmount(name: string) {
-  return numeric(name, { precision: 30, scale: 18, mode: "number" });
+  return numeric(name, { precision: 78, scale: 0, mode: "bigint" });
```

Comment:
Almost sure we need this change. We don't want to do rewards calculations on `number` which is very imprecise

Reply:
this method is only used by the staking-related tables, so whatever makes sense to you should work. later, we can consider reusing this method in the other tables and replace the `numeric(name, { precision: 30, scale: 18, mode: "number" });` definitions.